src/rtf-plugins/fixture-managers/yarpmanager/CMakeLists.txt

```diff
@@ -9,12 +9,12 @@ include_directories(SYSTEM ${RTF_INCLUDE_DIRS})
 
 add_definitions(-DSHLIBPP_FILTER_API)
 
-add_library(rtf_fixturemanager_yarpmanager MODULE YarpFixManager.h
-                                                  YarpFixManager.cpp)
-set_property(TARGET rtf_fixturemanager_yarpmanager PROPERTY PREFIX "")
-set_property(TARGET rtf_fixturemanager_yarpmanager PROPERTY OUTPUT_NAME yarpmanager)
+rtf_add_plugin(rtf_fixturemanager_yarpmanager YarpFixManager.h
+                                              YarpFixManager.cpp
+                                              OUTPUT_NAME yarpmanager)
```

Comment:
Didn't we add a `SOURCES` argument or something similar? I really don't like it in this way, the alignment is weird... 🤢

Reply:
No there is not a `SOURCES` argument, they are taken as unparsed_arguments :disappointed: 
Do you think I have to change it in RTF? But in this way I have to bump another time the patch number.